### DISCUSSION TITLE
Periodic video round-ups

### DISCUSSION THREAD

**Discussion by eidolonnight** [WordPress/developer-blog-content#142] [Topic Ideas]
Many camera and tech blogs do frequent video roundups, posts that showcase a collection of videos related to a particular topic or period. Here is an example: [Sony 50mm f/1.4 GM Youtube videos roundup](https://www.sonyalpharumors.com/sony-50mm-f-1-4-gm-youtube-videos-roundup-2/).

The Developer Blog could similarly highlight WordPress videos, such as these recent videos discussing aspects of 6.3:

- https://www.youtube.com/watch?v=OAwDPvaScrQ
- https://www.youtube.com/watch?v=s1L28GuA-y0
- https://www.youtube.com/watch?v=cCRX6XsK_ac

These roundups could even be built as playlists on YouTube and shared across social platforms. This would provide developers with n easy way to consume the content and promote the broader ecosystem of WordPress content creators.

**Comment by bph** [comment]
@eidolonnight Thank you so much for this suggestion. This came up in [last week's Editorial Group Meeting ](https://wordpress.slack.com/archives/C03RL47B3S8/p1688651767055339)in the core-dev-blog channel. 

Here are the bullet points: 

-  Writers already link to videos on WordPressTV if they fit the topic and reference summaries on Make blogs.
- The monthly roundups are already quite long (see [What's New for Developer 2023](https://developer.wordpress.org/news/2023/07/whats-new-for-developers-july-2023/)). Again, video links are added if they add value 
- Linking to personal YouTube spaces would violate external link policies. 
- A video roundup sounds like a good idea if it’s within the WordPress organization. 
- @eidolonnight Would you have bandwidth to take the helm on doing these pretty consistently? Or would you know someone?

**Reply by eidolonnight** [reply]
Awesome! Thanks for giving this some time and thought.

> The monthly roundups are already quite long

I agree. Adding videos to those is too much. This idea would be best as its own series/post. Even from a viewing standpoint, it's nice to have a post with _just_ videos. That way you can make some popcorn, sit back, and just watch.

> Linking to personal YouTube spaces would violate external link policies.

The [policy](https://developer.wordpress.org/news/tips-and-guidelines-for-writers/) states that "you should **prefer** WordPress-affiliated or open-source sites over third-party commercial sites", not that external links are not allowed entirely. I assume we would include a link to [the official WordPress YouTube channel](https://www.youtube.com/wordpress) and relevant [Learn](https://learn.wordpress.org/) videos right at the top of these round-ups, but this would also be an opportunity to highlight other creators and provide various perspectives on a given topic.

It's the variety of personalities and views that make these round-ups interesting. Only using video from within the WordPress organization would make this idea far less compelling.

> Would you have bandwidth to take the helm on doing these pretty consistently? Or would you know someone?

I wanted to pitch the idea and see if there was any interest before digging too deep. I'm happy to help, but it'd be really cool if we could gather some folks actively engaged in the creator ecosystem, like @metalandcoffee who [spoke about live streaming at WCUS 2022](https://wordpress.tv/2022/12/15/how-live-streaming-can-level-up-your-career/) and @ryanwelcher who regularly posts new [YouTube videos](https://www.youtube.com/@RyanWelcherCodes/videos).

@bph if you can confirm that the external link policy won't be an issue, then I can share this idea with the Marketing team and others to see who would be interested.

**Reply by bph** [reply]
For now the external link policy will be the issue if this idea includes non-WordPress profiles on various services. 
On the developer blog we don't allow links to Ryan's twitch or YouTube space nor of Gutenberg Times for that matter. 

Most influencers make money with affiliate links to products. The group working on the developer blog doesn't have the bandwidth to make sure those links are acceptable regarding GPL license, code of conduct. Linking to outside places that are not on the allowed list, is just too much of a risk to experience abuse.

**Reply by eidolonnight** [reply]
Thanks for the clarification. I guess this particular idea is dead in the water.

The recommendation to "link to videos on WordPressTV if they fit the topic and reference summaries on Make blogs" is still good. We also already put together YouTube playlists for WordCamps and other topics. Those can continue, and we'll shelve this idea until there's a channel or opportunity to highlight other creators.
